scripts/get_users_meta_docs.js

```diff
@@ -0,0 +1,97 @@
+const inquirer = require('inquirer'),
+      PouchDB = require('pouchdb'),
+      fs = require('fs'),
+      path = require('path');
+
+if(process.argv[2] === '-h' || process.argv[2] === '--help') {
```

Comment:
You're just on the cusp of this arg parsing being too complicated for manual stuff. If you get more complicated than this I'd suggest looking at something like minimism.